networkx/algorithms/min_edge_cover.py

```diff
@@ -0,0 +1,63 @@
+#TODO: Add Test Cases
+"""
+An edge cover of a graph is a set of edges such that every vertex of
+the graph is incident to at least one edge of the set.
+A minimum edge cover is an edge covering of smallest possible size.
+
+https://en.wikipedia.org/wiki/Edge_cover
+
+"""
+import networkx as nx
+__all__ = ["min_edge_cover"]
+__author__ = """Nishant Nikhil (nishantiam@gmail.com)"""
```

Comment:
#1728 
@nishnik Even if NetworkX has this variable in many places due to [historical reasons](https://github.com/networkx/networkx/issues/1728#issuecomment-129481442), a more simple way is to put copyright lines on the top of file as comment like [this](https://github.com/networkx/networkx/blob/328d899fb17bac857eedd9ec3e03f4ede9b11c63/examples/basic/properties.py#L5-L10). Nevertheless it's a mere personal choice of convention to follow.